packages/rest/src/parser.ts

```diff
@@ -95,6 +92,16 @@ function buildOperationArguments(
 ): OperationArgs {
   const args: OperationArgs = [];
 
+  let requestBodyIndex: number = -1;
+  if (hasArgumentsFromBody(operationSpec)) {
+    const i = (<RequestBodyObject>operationSpec.requestBody)[
```

Comment:
`requestBody` could be `RequestBodyObject` or `ReferenceObject`, see: https://github.com/metadevpro/openapi3-ts/blob/master/src/model/OpenApi.ts#L99

I specify the type here to avoid resolving the `ReferenceObject`, I will add a comment here.